playground/backend/internal/environment/environment_service_test.go

```diff
@@ -78,15 +78,15 @@ func TestNewEnvironment(t *testing.T) {
 	}{
 		{name: "create env service with default envs", want: &Environment{
 			NetworkEnvs:     *NewNetworkEnvs(defaultIp, defaultPort, defaultProtocol),
-			BeamSdkEnvs:     *NewBeamEnvs(defaultSdk, executorConfig),
+			BeamSdkEnvs:     *NewBeamEnvs(defaultSdk, executorConfig, ""),
```

Comment:
Let's create a separate variable `preparedModDir := ""` and replace empty strings in `NewBeamEnvs` function